docs/components/dartpad/buttons/flat/main.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      home: MyHomePage(title: 'Flutter Demo Text Page'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            FlatButton(
+              textColor: Color(0xFF6200EE),
```

Comment:
why are we changing the textColor here and below?

Reply:
So it looks like the example images:

https://github.com/JoseAlba/material-components-flutter/blob/dartpad/docs/components/buttons.md

Reply:
Would it make sense to use a Theme instead of providing component specific visual properties?